iron-form.html

```diff
@@ -288,7 +295,8 @@
         // were already added as a custom element, they don't need
         // to be re-added.
         if (!this._useValue(el) ||
-            (el.hasAttribute('is') && json[el.name])) {
+            (el.hasAttribute('is') && json[el.name]) || 
```

Comment:
That's still potentially valid though. It's an edge case, but still a valid one. 

A form like

```
<paper-input name='name[]'></paper-input>
<paper-input name='name[]'></paper-input>
```

with first and last name being split on the server is a weird, but still valid solution.